libs/pmp-web/user/data-access/src/lib/+state/user.reducer.ts

```diff
@@ -0,0 +1,60 @@
+import { fromUserActions } from './user.actions';
+import { HttpErrorResponse } from '@angular/common/http';
+import { UserStatistics } from '@pimp-my-pr/shared/domain';
+
+export const USER_FEATURE_KEY = 'user';
+
+export interface UserState {
+  userStatisticsCollection: UserStatistics[];
```

Comment:
Why no entity adapter?

Reply:
If entity adapter will be used please add selectors from the adapter.